homeassistant/components/switcher_kis/__init__.py

```diff
@@ -134,13 +143,32 @@ async def async_set_auto_off_service(service: ServiceCallType) -> None:
             ) as swapi:
                 await swapi.set_auto_shutdown(service.data[CONF_AUTO_OFF])
 
+        async def async_turn_on_with_timer_service(service: ServiceCallType) -> None:
+            """Use for handling setting device auto-off service calls."""
+
+            await _validate_edit_permission(
+                hass, service.context, service.data[CONF_ENTITY_ID]
```

Comment:
We need to update the constant here too.

Reply:
Fixed all places